internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -403,21 +403,21 @@ func (r *Reconciler) createMachineDeployment(ctx context.Context, cluster *clust
 		cluster: cluster,
 		desired: md.InfrastructureMachineTemplate,
 	}); err != nil {
-		return errors.Wrapf(err, "failed to create %s", tlog.KObj{Obj: md.Object})
+		return errors.Wrapf(err, "failed to create %s, %s", md.Object.Kind, err)
```

Comment:
should this be:
```suggestion
		return errors.Wrapf(err, "failed to create %s", md.Object.Kind)
```
otherwise we are reporting twice error (same below)